packages/sol-rayz/src/getParsedNftAccountsByCollection.ts

```diff
@@ -0,0 +1,33 @@
+// @ts-nocheck
```

Comment:
please don't do such things

Reply:
Again. You used it previously; it's unfair to enforce using ts only in some PRs)